lib/routes/emails.js

```diff
@@ -174,6 +176,9 @@ module.exports = (log, db, mailer, config, customs, push) => {
               flowId,
               flowBeginTime,
               service,
+              ip,
+              location: geoData.location,
+              timeZone: geoData.timeZone,
```

Comment:
Worth adding assertions for these properties in the local route test?

https://github.com/mozilla/fxa-auth-server/blob/ab17bf85fe0eb269b422c44becf560244d41334a/test/local/routes/emails.js#L307-L316